src/auth.ts

```diff
@@ -16,6 +17,10 @@ export async function configureAuthentication() {
   const id = core.getInput(constants.INPUT_SERVER_ID);
   const username = core.getInput(constants.INPUT_SERVER_USERNAME);
   const password = core.getInput(constants.INPUT_SERVER_PASSWORD);
+  const settingsDirectory =
+    core.getInput(constants.INPUT_SETTINGS_PATH) || path.join(os.homedir(), M2_DIR);
+  // Consider setting overwriteSettings = false by default in next major release
```

Comment:
This comment can be removed given that we're keeping it to `true` for this release

Reply:
I am mostly added it for V3 :) but I can remove

Reply:
`v3` might be a long long time out so that's why I'm inclined to just remove it now 😄 Setting it to `false` in the future would also break the easy setup we have with GitHub package registry so I don't think down the line that will be an easy choice that is made

Reply:
Removed